src/utils/create-export-document.js

```diff
@@ -18,9 +18,41 @@ const colors = {
  *   terms
  */
 function createBaseDocument() {
+  const url = window.location.origin + window.location.pathname;
+
   return {
+    header: {
+      style: 'pageHeaderOrFooter',
+      columns: [
+        { text: `Phenotypr Body: ${url}`, link: url, style: 'link' },
+        { text: new Date().toLocaleDateString(), alignment: 'right' }
+      ]
+    },
+
+    footer: (currentPage, pageCount) => {
+      return {
+        text: `Page ${currentPage} of ${pageCount}`,
+        style: 'pageHeaderOrFooter',
+        alignment: 'center'
+      };
+    },
+
     content: [
-      { text: 'Your HPO Terms', style: 'heading' },
+      {
+        text: 'Your HPO Terms',
+        style: 'heading'
+      },
+      '\n',
+      {
+        text: [
+          'The table below shows the symptoms you selected using the ',
+          { text: 'Phenotypr Body tool', link: url, style: 'link' },
+          ' and the corresponding HPO terms. Knowing you or your child\'s HPO mapping can ',
+          'provide your healthcare provider access to quality, structured phenotyping data. ',
+          'Providing this information to your clinicians can assist in diagnosis (e.g., ',
+          'there may be something your medical team doesn\'t know about!).'
+        ]
+      },
```

Comment:
It seems like the newlines in the content list should be the responsibility of the renderer to add (`contents.join('\n')`). I think the replacement of the TABLE_PLACEHOLDER is okay for documenting intent, but it would probably be equally clear to just push the table onto the end of the content list.

Reply:
The renderer is a very thin wrapper around pdfmake, which seems to take the stance that it's the job of the document structure to say where line breaks are (see the [pdfmake playground](http://pdfmake.org/playground.html) for lots of examples). I'll combine the newlines with the document nodes, which will at least keep everything together.

I can definitely remove the TABLE_PLACEHOLDER code at this point and just use push. The placeholder only exists because I thought there might be content on both sides of the table, which has turned out not to be true.